app/src/main/res/layout/screen_teleconsult_success.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="utf-8"?>
+<org.simple.clinic.teleconsultlog.success.TeleConsultSuccessScreen xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:background="@color/grey4">
+
+  <RelativeLayout
```

Comment:
Is there a reason we are creating a separate layout here instead of using the `Toolbar` component?